hibernate/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot'
+	id 'org.springframework.aot.smoke-test'
+	id 'org.graalvm.buildtools.native'
+}
+
+dependencies {
+	implementation(platform(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES))
+	implementation("org.springframework.boot:spring-boot-starter-jdbc")
+	implementation("org.springframework:spring-orm")
+	implementation("jakarta.persistence:jakarta.persistence-api")
+	implementation("org.hibernate:hibernate-core:6.1.2.Final")
```

Comment:
Isn't that version managed by Boot?

Reply:
Boot manages `org.hibernate.orm:hibernate-core`. The group ID is different. Are these two different JARs?

Reply:
doh my bad - should have looked at the `3.0.0-SNAPSHOT` dependencies, not current 😅 - feel free to alter at your convenience